messages/Config.js

```diff
@@ -3,6 +3,8 @@ module.exports = {
 	"InvalidStringArrayValue": "Specify a valid property for '%s' in %s for %s. Supply an array of strings: %s.",
 	"InvalidBooleanValue": "Specify a valid property for '%s' in %s for %s. Supply a boolean: %s.",
 	"InvalidNumberValue": "Specify a valid property for '%s' in %s for %s. Supply a number: %s.",
+	"InvalidStringValue": "Specify a valid property for '%s' in %s for %s. Supply a string: %s.",
+	"InvalidStringEnumValue": `Specify a valid property for '%s' in %s for %s. Valid options include: %s.`,
```

Comment:
@teresa-allen-sfdc , these messages need to be reviewed. The first is displayed when a config property that is expected to be a string is something other than a string, and in the wild it will look like this:
```Specify a valid property for 'someproperty' in /path/to/Config.json for SomeEngine. Supply a string: currentNonStringValue.```
 The second is displayed when a config property that is expected to be one of a small set of options is something other than one of those options. It will look like this in the wild:
```Specify a valid property for 'someProperty' in /path/to/Config.json for SomeEngine. Valid options include: "every"|"valid"|"option"|"formatted"|"like"|"this".```